### ISSUE TITLE
False negative of LogicalAssignment with --checkFeatures

### ISSUE THREAD

**Issue by p3k** [yowainwright/es-check#361]
Run `es-check` with this minified file:

```js
"use strict";(self.webpackChunk$wlportal=self.webpackChunk$wlportal||[]).push([[422],{35904:(e,d,r)=>{r.r(d),r.d(d,{default:()=>s});var a=r(71996),t=r(75915),l=r(20467);let i={components:{Foo:t.A,Bar:l.A},props:{data:{type:Object,default:()=>({})},foo:{type:Object,default:()=>({})}},setup:()=>({})},s=(0,r(1877).A)(i,[["render",function(e,d,r,t,l,i){let s=(0,a.g2)("Bar"),p=(0,a.g2)("Foo");return(0,a.uX)(),(0,a.CE)("div",null,[(0,a.bF)(s,{data:r.data,header:r.foo.slider.header,boundaries:r.foo.slider.boundaries,value:r.foo.slider.value,dragUpdate:r.foo.slider.dragUpdate,callback:r.foo.slider.callback,width:r.foo.slider.width},null,8,["data","header","boundaries","value","dragUpdate","callback","width"]),(0,a.bF)(s,{header:r.foo.sliderTransparency.header,boundaries:r.foo.sliderTransparency.boundaries,value:r.foo.sliderTransparency.value,dragUpdate:r.foo.sliderTransparency.dragUpdate,callback:r.foo.sliderTransparency.callback,width:r.foo.sliderTransparency.width},null,8,["header","boundaries","value","dragUpdate","callback","width"]),r.foo.menu&&0!==Object.keys(r.foo.menu).length?((0,a.uX)(),(0,a.Wv)(p,{key:0,header:r.foo.menu.header,toggle:r.foo.menu.toggle,items:r.foo.menu.items},null,8,["header","toggle","items"])):(0,a.Q3)("",!0)])}]])}}]);
```

```sh
$ es-check es2019 test.min.js --checkFeatures --checkForPolyfills
info: ES-Check: checking 1 file...
/home/foo/node_modules/es-check/lib/detectFeatures.js:61
    const error = new Error(
                  ^

Error: Unsupported features detected: LogicalAssignment. These require a higher ES version than 10.
    at detectFeatures (/home/foo/node_modules/es-check/lib/detectFeatures.js:61:19)
    at processFullAST (/home/foo/node_modules/es-check/lib/check-runner/utils.js:326:50)
    at /home/foo/node_modules/es-check/lib/check-runner/utils.js:406:12
    at async Promise.all (index 0)
    at async processBatchedFiles (/home/foo/node_modules/es-check/lib/helpers/files.js:7:12)
    at async processConfig (/home/foo/node_modules/es-check/lib/check-runner/index.js:137:19)
    at async runChecks (/home/foo/node_modules/es-check/lib/check-runner/index.js:165:20) {
  type: 'ES-Check',
  features: [ 'LogicalAssignment' ],
  ecmaVersion: 10
}

Node.js v22.21.1
```

Grepping the file I cannot find any instances of `&&=`, `||=` or `??=`:

```sh
$ grep -E '[&|?]{2}[[:space:]]*=' test.min.js
# (zilch!)
```
